ydb/core/blobstorage/dsproxy/dsproxy_put.cpp

```diff
@@ -129,12 +130,13 @@ class TBlobStorageGroupPutRequest : public TBlobStorageGroupRequestActor<TBlobSt
     }
 
     void Accelerate() {
-        if (IsAccelerated) {
+        if (AccelerateRequestsSent == 2) {
             return;
         }
-        IsAccelerated = true;
+        ++AccelerateRequestsSent;
         Action(true);
 //        *(IsMultiPutMode ? Mon->NodeMon->AccelerateEvVMultiPutCount : Mon->NodeMon->AccelerateEvVPutCount) += v.size();
+        AccelerateIfNeeded();
```

Comment:
Action(true) a few lines above already accelerates, no?

Reply:
Yes, but we need to schedule second accelerate in case current acceleration sends request to slow disk and doesn't recieve response in time. This call schedules next acceleration.